tests/ui/cast.stderr

```diff
@@ -363,6 +363,15 @@ help: ... or use `try_from` and handle the error accordingly
 LL |     i8::try_from((-99999999999i64).min(1));
    |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+error: `(-99999999999i64)` is never greater than `1` and has therefore no effect
+  --> tests/ui/cast.rs:200:5
+   |
+LL |     (-99999999999i64).min(1) as i8;
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `(-99999999999i64)`
+   |
+   = note: `-D clippy::unnecessary-min-max` implied by `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::unnecessary_min_max)]`
+
```

Comment:
Can add this lint to the `allow` in `tests/ui/cast.rs` since it's not relevant to that test file